src/test/java/io/pravega/connectors/hadoop/PravegaInputFormatTest.java

```diff
@@ -105,6 +107,16 @@ private ClientFactory mockClientFactory() {
         return mockClientFactory;
     }
 
+    private StreamManager mockStreamManager() {
+        StreamManager streamManager = mock(StreamManager.class);
+        io.pravega.client.admin.StreamInfo streamInfo = mock(io.pravega.client.admin.StreamInfo.class);
```

Comment:
Hopefully, in the future we may work only with one `StreamInfo` class, so we will not have this import problem.